### PR TITLE
Add Home button in footer navigation bar

### LINKED ISSUE #119: Implement a Home button for easier navigation
This might be best implemented as a sticky footer related to #121

### PR DESCRIPTION
**Pull request by mddilley**
Closes #119

### CODE REVIEW

**Comment by mddilley** [comment]
@mateoclarke I pulled #121 into this branch to make sure they played well together. Let me know if there is anything else you would like to in footers.

File: src/components/Home.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ const mainPages = [
 ];
 
 class Home extends Component {
+  debugger;
```
Comment (mateoclarke) [inline src/components/Home.js]: 🐛

File: src/components/Shared/NavFooter.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  },
+  {
+    title: "All Work Orders",
+    link: "/all-work-orders",
+    icon: faTruck,
+  },
+  {
+    title: "New Work Order",
+    link: "/work-order/new",
+    icon: faWrench,
```
Comment (mateoclarke) [inline src/components/Shared/NavFooter.js]: maybe this is a plus sign instead of a wrench?

File: src/components/Shared/NavFooter.js [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  },
+];
+
+const linkColors = {
+  inactive: "#6C757C",
+  active: "#307BFF",
```
Comment (mateoclarke) [inline src/components/Shared/NavFooter.js]: not necessary right now, but we could put all the colors we use in a constants file.

File: src/components/Home.js (hunk 1 above)
Comment (mddilley) [reply src/components/Home.js]: 🤦‍♂... Removed!

File: src/components/Shared/NavFooter.js (hunk 2 above)
Comment (mddilley) [reply src/components/Shared/NavFooter.js]: I tried it out, and it looks good with the plus sign! Maybe we should change all of the New Work Order icons to plus signs? Merging for now so we think about it.
Screen Shot 2019-07-24 at 11 57 17 AM

File: src/components/Shared/NavFooter.js (hunk 3 above)
Comment (mddilley) [reply src/components/Shared/NavFooter.js]: Yep - will make any color/themes changes easier in the long run.

File: src/components/Shared/NavFooter.js (hunk 2 above)
Comment (mateoclarke) [reply src/components/Shared/NavFooter.js]: yeah I like that, I'll add it as an issue.

**Pull request merged**